scripts/clean.sh

```diff
@@ -0,0 +1,7 @@
+cp ../records.json /tmp/records.json.bak
```

Comment:
Should probably put this as another target in the Makefile instead of a standalone script.